azkaban-exec-server/src/main/java/azkaban/execapp/FlowRunner.java

```diff
@@ -602,9 +604,14 @@ private void finalizeFlow(final ExecutableFlowBase flow) {
     final long durationSec = (flow.getEndTime() - flow.getStartTime()) / 1000;
     switch (flow.getStatus()) {
       case FAILED_FINISHING:
-        this.logger.info("Setting flow '" + id + "' status to FAILED in "
+        Status status = isKilled() ? Status.KILLED : Status.FAILED;
+        this.logger.info("Setting flow '" + id + "' status to "+status+" in "
```

Comment:
Nope, I didn't have it activated. I'm a bit afraid to activate save plugin because not all code is yet formatted (or is it now?), and if a previously unformatted class gets auto-formatted when I have also some manual changes in that file, I'm in trouble :) Usually I take care of formatting any way, this is plain mistake, sorry..